backend/calendar/user_events.py

```diff
@@ -1,71 +1,177 @@
+            event_desc is TEXT, not NULL
+            event_type is VARCHAR(50), not NULL
+            event_title is VARCHAR(50), not NULL
+            start_epoch is INT, NULLABLE
+            end_epoch is INT, NULLABLE
+            on_to_do_list is BOOLEAN, not NULL
+            extra_data is VARCHAR(250), NULLABLE
+            is_submitted is BOOLEAN, NULLABLE
+            want_notification is BOOLEAN, NULLABLE
+
+        postcondition:
+            inputs given values into DB, assiging event to usr_id
+            returns error if usr_id not found
+            returns success message if successful 
+
+    '''
+
     if(usr_id_exists(usr_id)):
+        created_epoch = datetime.now().timestamp()
+        next_id = get_event_id()
         con = sqlite3.connect('server/usrDatabase/usrDB.db')
         cursor = con.cursor()
         cursor.execute('''
             INSERT INTO calendar (usr_id, event_id, created_epoch, event_desc, event_type,
                                 event_title, start_epoch, end_epoch, on_to_do_list, extra_data, 
                                 is_submitted, want_notification)
             VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)
-        ''', (usr_id, event_id, created_epoch, event_desc, event_type,
+        ''', (usr_id, next_id, created_epoch, event_desc, event_type,
             event_title, start_epoch, end_epoch, on_to_do_list, extra_data,
             is_submitted, want_notification))
         
         con.commit()
         con.close()
+        return jsonify({'result': 'event created'})
     else:
-        raise UserNotFoundException
+        return jsonify({'result': 'id not found'})
+
+
+def get_event(event_id):
+
+    '''
+        precondition:
+            event_id is in DB as INT
+
+        postcondition:
+            returns event information
+            returns error if event_id does not exist
+    '''
+
 
-def read_event(event_id):
     if event_id_exists(event_id):
         con = sqlite3.connect('server/usrDatabase/usrDB.db')
         cursor = con.cursor()
         cursor.execute('SELECT * FROM calendar WHERE event_id = ?', (event_id,))
-        event = cursor.fetchone()
+        result = cursor.fetchone()
+        event = result[0]
         con.close()
-        return event
+        return jsonify({'result': event})
     else:
-        raise EventNotFoundException
+        return jsonify({'result': 'event not found'})
+
+def get_usr_events(usr_id):
+
+    '''
+        precondition:
+            usr_id is in DB as INT
+
+        postcondition:
+            returns all event information
+            returns error if usr_id does not exist
+    '''
 
-def read_usr_events(usr_id):
     if usr_id_exists(usr_id):
         con = sqlite3.connect('server/usrDatabase/usrDB.db')
         cursor = con.cursor()
         cursor.execute('SELECT * FROM calendar WHERE usr_id = ?', (usr_id,))
-        events = cursor.fetchall()
+        result = cursor.fetchall()
+        events = result[0]
```

Comment:
same here as above, not sure how fetchall() works and what kind of array it returns